js/app/modules/arrangement/accordion.js

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 Endless Mobile, Inc.
+
+/* exported Accordion */
+
+const Gtk = imports.gi.Gtk;
+
+const Arrangement = imports.app.interfaces.arrangement;
+const Module = imports.app.interfaces.module;
+
+const TRANSITION_DURATION = 500;
+
+/**
+ * Class: Accordion
+ * Arrangement which puts its cards into an accordion-like widget.
+ * Each segment of the accordion has a title, which is always visible,
+ * and a revealer, in which lives the actual card. Clicking on the title
+ * expands the corresponding revealer.
+ */
+const Accordion = new Module.Class({
+    Name: 'Arrangement.Accordion',
+    Extends: Gtk.Grid,
+    Implements: [Arrangement.Arrangement],
+
+    _init: function (props={}) {
+        props.orientation = Gtk.Orientation.VERTICAL;
+        props.valign = Gtk.Align.START;
+        this.parent(props);
+    },
+
+    // Arrangement override
+    pack_card: function (card) {
+        let label = new Gtk.Button({
```

Comment:
Feels like this would be nicer if this too was a card. By default you'd want title card here, but you could also use a similar small card in the future. And you'd get capitalization, highlight and before after decoration support that just GtkButton wont buy you.

Pretty sure having two card types would break the arrangement interface, so probably not a change for this specific PR. More just thinking aloud here...